File: SiteScraper/gamemodel.py
class Game:
    def __init__(self):
        self.id = None
        self.name = None
        self.cartclass = None
        self.catalogentry = None
        self.players = None
        self.releasedate = None
        self.peripherals = None
        self.peripheralsImage = None

    def to_dict(self):
        return {
            "id": self.id,
            "name": self.name,
            "cartclass": self.cartclass,
            "catalogentry": self.catalogentry,
            "players": self.players,
            "releasedate": self.releasedate,
            "peripherals": self.peripherals,
            "peripheralsImage": self.peripheralsImage
        }

File: CartDB.Parser/Models/GameModel.cs
namespace CartDB.Parser.Models
{
    public class GameModel
    {
        public string Name { get; set; }
        public string CartClass { get; set; }
        public string CatalogEntry { get; set; }
        public string Players { get; set; }
        public string ReleaseDate { get; set; }
        public string Peripherals { get; set; }
        public string PeripheralsImage { get; set; }
        public DeveloperModel Developer { get; set; }
        public PublisherModel Publisher { get; set; }
        public RegionModel Region { get; set; }
    }
}
